democracy/views/hearing.py

```diff
@@ -134,32 +133,39 @@ def get_queryset(self):
                 to_attr='main_section_list'
             )
         )
+
+        if not self.request.user.is_superuser:
```

Comment:
In general I could do more commenting I give you that, but I don't really understand this one, the two lines of code are saying just the same thing? :D